lib/pollynomial/synthesizer.rb

```diff
@@ -38,9 +38,8 @@ def synthesize(text, file_name: "tmp.mp3")
       end
     end
 
-    def available_voices_in(language_code: 'en-US')
-      voices = client.describe_voices(language_code: language_code)
-      voices.voices if voices
+    def voices(language=nil)
```

Comment:
It kinda is.... since it now allows all languages (i.e. accept no arguments), you end up with a broken sentence: `synth.available_voices_in`...  in what?